fix(scss): content layout migration grid template

PiiXiieeS opened the pull request:
- due to issues with flex-basis let's try to move to grid template to make
the layout maintenance easier
- iadc theme is also aligned to use grid-template

PiiXiieeS committed:
fix(scss): content layout migration grid template

- due to isses with flex-basis let's try to move to grid template to make
the layout maintenance easier
-

PiiXiieeS committed:
fix(scss): content layout migration grid template

- due to isses with flex-basis let's try to move to grid template to make
the layout maintenance easier
-

PiiXiieeS committed:
fix: added rest of the layouts as grid template

Yet work in progress and while visualizaiton is improved not yet optimal

PiiXiieeS committed:
fix: added rest of the layouts as grid template

Yet work in progress and while visualizaiton is improved not yet optimal

Zougouda committed:
fix: WEB-49 Added a box-sizing attribute to video & img tags to prevent horizontal overflow caused by padding rules on these tags

Zougouda committed:
fix: WEB-49 Added a box-sizing attribute to video & img tags to prevent horizontal overflow caused by padding rules on these tags

kiru42 committed:
Merge branch 'main' into bugfix/content-layout-multi-column

Zougouda committed:
fix: WEB-49 removed debug colors

Zougouda committed:
fix: WEB-49 removed debug colors

Zougouda committed:
Merge branch 'bugfix/content-layout-multi-column' of https://github.com/Sanofi-IADC/konviw into bugfix/content-layout-multi-column

PiiXiieeS commented:
@kiru42, here I believe we will have to make some cypress test adjustments, right?

kiru42 committed:
fix: caching key for npm packages

kiru42 committed:
fix: caching key for npm packages

kiru42 committed:
fix: health check

kiru42 committed:
fix: health check

kiru42 committed:
fix: ci remove nodejs 15 which is unsupported

kiru42 committed:
fix: ci remove nodejs 15 which is unsupported

kiru42 committed:
fix: jest timeout

kiru42 committed:
fix: jest timeout

kiru42 committed:
fix: jest timeout

kiru42 committed:
fix: jest timeout

kiru42 committed:
fix: jest timeout

kiru42 committed:
fix: jest timeout

kiru42 commented:
> @kiru42, here I believe we will have to make some cypress test adjustments, right?

Yes you are right, the test will probably fail after the merge. Then we will have to update the screenshots.

PiiXiieeS committed:
fix(scss): align iadc theme with default

- content-layout aligned to use grid-template
- add missing macro-jira.scss in iadc template

PiiXiieeS committed:
fix(scss): align iadc theme with default

- content-layout aligned to use grid-template
- add missing macro-jira.scss in iadc template

PiiXiieeS committed:
fix: remove uncommented fixColGroupWidth

PiiXiieeS committed:
fix: remove uncommented fixColGroupWidth

Zougouda committed:
fix: WEB-49 Fixed arrays width and word-breaks in headers

Zougouda committed:
fix: WEB-49 Fixed arrays width and word-breaks in headers

Zougouda committed:
fix: WEB-49 Fixed various table overflows in both desktop and mobile versions

Zougouda committed:
fix: WEB-49 Fixed various table overflows in both desktop and mobile versions

Zougouda committed:
fix: WEB-49 Now using grid display instead of flex (flew is still used as columns in mobile mode)

Zougouda committed:
fix: WEB-49 Now using grid display instead of flex (flew is still used as columns in mobile mode)

kiru42 reviewed (approved):
LGTM.

Zougouda committed:
fix: WEB-49 fixed-width updated from 900px to 1200px

Zougouda committed:
fix: WEB-49 fixed-width updated from 900px to 1200px

PiiXiieeS committed:
fix: align iadc style with default and remove comments

PiiXiieeS committed:
fix: align iadc style with default and remove comments

PiiXiieeS committed:
build: fix path start:prod

PiiXiieeS committed:
build: fix path start:prod

kiru42 committed:
fix: fix vercel

kiru42 committed:
fix: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix vercel

kiru42 committed:
build: fix cypress tests

kiru42 committed:
build: fix cypress tests

kiru42 committed:
fix: undefined match error

kiru42 committed:
fix: undefined match error

kiru42 committed:
style: fix newline

kiru42 committed:
style: fix newline

kiru42 committed:
test: fix cypress snapshot

kiru42 committed:
test: fix cypress snapshot

kiru42 committed:
test: fix cypress snapshot

kiru42 committed:
test: fix cypress snapshot

kiru42 committed:
test: remove default config for vercel

kiru42 committed:
test: remove default config for vercel

kiru42 committed:
test: add serverless function for vercel

kiru42 committed:
test: add serverless function for vercel

kiru42 committed:
test: add serverless function for vercel

kiru42 committed:
test: add serverless function for vercel

kiru42 committed:
fix: vercel build

kiru42 committed:
fix: vercel build

kiru42 committed:
fix: vercel build

kiru42 committed:
fix: vercel build

kiru42 committed:
fix: vercel build

kiru42 committed:
fix: vercel build

kiru42 committed:
fix: vercel build

kiru42 committed:
fix: vercel build

kiru42 committed:
fix: vercel build

kiru42 committed:
fix: vercel build

kiru42 committed:
fix: vercel build

kiru42 committed:
fix: vercel build

kiru42 committed:
fix: vercel build - rollback

kiru42 committed:
fix: vercel build - rollback

kiru42 committed:
ci: update cypress workflow for vercel preview deployment

kiru42 committed:
ci: update cypress workflow for vercel preview deployment

kiru42 committed:
ci: fix typo for  cypress workflow

kiru42 committed:
ci: fix typo for  cypress workflow

kiru42 committed:
Merge branch 'bugfix/content-layout-multi-column' of github.com:Sanofi-IADC/konviw into bugfix/content-layout-multi-column

kiru42 committed:
fix: gitignore
